gap/Parser.gi

```diff
@@ -420,8 +420,8 @@ InstallGlobalFunction( AutoDoc_Parser_ReadFiles,
         od;
         return example_node;
     end;
-    read_session_example := function( is_tested_example )
-        local temp_string_list, temp_curr_line, temp_pos_comment, is_following_line, item_temp, example_node;
+    read_session_example := function( is_tested_example, pt_mode )
```

Comment:
pt_mode = plaintext_mode?

If so, please rename it, or at least add a comment here that states what it is. This function is already cryptic enough, no need to make it worse ;-)